eos/statistics/log-likelihood.cc

```diff
@@ -1057,13 +1057,22 @@ namespace eos
         struct UniformBoundBlock :
             public LogLikelihoodBlock
         {
-            ObservableCache cache;
+            ObservableCache _cache;
```

Comment:
This is misleading. You prepend the names with an underscore, but the variables are publicly accessible. (Which is perfectly fine! This class is only exposed internally.)

Can you undo the renaming and remove underscores from the new variables?

Reply:
Sure! This issue is also present in the `MultivariateGaussianBlock`, we may want to change that in the future.